compose/cli/main.py

```diff
@@ -308,13 +308,24 @@ def logs(self, project, options):
         Usage: logs [options] [SERVICE...]
 
         Options:
-            --no-color  Produce monochrome output.
+            --no-color          Produce monochrome output.
+            -f, --follow        Follow log output.
+            -t, --timestamps    Show timestamps.
+            --tail="all"        Number of lines to show from the end of the logs
+                                for each container.
         """
         containers = project.containers(service_names=options['SERVICE'], stopped=True)
 
         monochrome = options['--no-color']
+        log_args = {}
+        log_args['stream'] = options['--follow']
```

Comment:
I understand the difference between `stream` and `follow`, but I'm not sure to understand the advantage of this change.

If you set `follow` to `True`, should `stream` always be `True` ?

Maybe should we open an issue on `docker-py` to talk about it before to merge this PR ?